src/drivers/middleware/jwt.config.ts

```diff
@@ -38,6 +38,7 @@ export const enforceTokenAccess = jwt({
     '/outages',
     /\/users\/[0-z,.,-]+\/cards/i,
     '/library/stats',
+    { url: /\/learning-objects\/[0-z,.,-,%]\/[0-z,.,-,%]/, methods: ['GET'] },
```

Comment:
Can you explain this line? Do we want every route with /learning-objects to be unauthenticated? If so, we should remove the authentication logic from the Gateway instead of bypassing it.